operator/v1/types_network.go

```diff
@@ -252,13 +252,18 @@ type KuryrConfig struct {
 	OpenStackServiceNetwork string `json:"openStackServiceNetwork,omitempty"`
 }
 
-// ovnKubernetesConfig is the proposed configuration parameters for networks
+// ovnKubernetesConfig contains the configuration parameters for networks
 // using the ovn-kubernetes network project
 type OVNKubernetesConfig struct {
 	// mtu is the MTU to use for the tunnel interface. This must be 100
 	// bytes smaller than the uplink mtu.
 	// Default is 1400
+	// +optional
 	MTU *uint32 `json:"mtu,omitempty"`
+	// geneve port is the USB port to be used by geneve encapulation
+	// Default is 6081
+	// +optional
+	GenevePort *uint32 `json:"geneve-port,omitempty"`
```

Comment:
does this affect the port that is allowed in firewall rule in the installer.

currently we open the sdn and ovn ports (seem like they are the defaults) in the installer as the network operator cannot manage them as of today.

So If we allow users to modify this port, what is the expectation
a) the installer will need to be update to use this _changeable_ port 
^^ I don't that is a great idea as it puts a lot of the decision into installer and we would not like to do it.
b) the network-operator would have the capability to realize this modification.

imo (b) should be the requirement for accepting this api.

Reply:
The default for the geneve port is 6081 we would like to change it to 9081.

Reply:
@squeed can you answer @abhinavdahiya question? CC: @dcbw